scripts/install.sh

```diff
@@ -69,31 +77,107 @@ printf "Network: $NETWORK\n"
 
 cd $KEEP_BEACON_SOL_PATH
 
-printf "${LOG_START}Installing YARN dependencies...${LOG_END}"
-yarn install
+printf "${LOG_START}Installing beacon YARN dependencies...${LOG_END}"
+yarn
 
 if [ "$NETWORK" == "development" ]; then
     printf "${LOG_START}Unlocking ethereum accounts...${LOG_END}"
     KEEP_ETHEREUM_PASSWORD=$KEEP_ETHEREUM_PASSWORD \
         npx hardhat unlock-accounts --network $NETWORK
 fi
 
-printf "${LOG_START}Building contracts...${LOG_END}"
-yarn clean
-yarn build
+if [ "$SKIP_DEPLOYMENT" != true ]; then
 
-if [ "$SKIP_DEPLOYMENT" = false ] ; then
-  printf "${LOG_START}Deploying contracts...${LOG_END}"
+  # create tmp/ dir for fresh installations
+  rm -rf $TMP && mkdir $TMP && cd $TMP
 
-  npx hardhat deploy --reset --export export.json --network $NETWORK
-fi
+  # clone solidity-contracts as a dependency for beacon, ecdsa and tbtc
+  git clone https://github.com/threshold-network/solidity-contracts.git
+  cd solidity-contracts
+
+  printf "${LOG_START}Building solidity-contracts...${LOG_END}"
```

Comment:
```suggestion
  printf "${LOG_START}Building threshold-network contracts...${LOG_END}"
```

Reply:
[6b306c4](https://github.com/keep-network/keep-core/pull/3096/commits/6b306c4eee00dd84e0cd81c9d3c083c9e81a72c4) Both versions are kind of vague. Threshold-network is the name of the whole ecosystem, not that specific project. I merged them together.